pipeline-runner/R/qc-6-integrate_scdata-harmony.R

```diff
@@ -68,7 +68,7 @@ run_harmony <- function(scdata_list, config, cells_id) {
       harmony::RunHarmony(
         scdata,
         group.by.vars = "samples",
-        reduction = "pca_for_harmony",
+        reduction.use = "pca_for_harmony",
         dims.use = 1:npcs
```

Comment:
seurat v5 now supports harmony integration out of the box, that could be implemented later.

Reply:
in addition, the bug the the number of pcs used was fixed in a harmony update some time back, so the "pca_for_harmony" workaround should no longer be necessary. not for this PR though.

Reply:
Sounds good -- yeah there are some other things that can be updated as well (`presto` is used by default for wilcox is Seurat FindMarkers), but this PR is already massive and has lots of room for introducing bugs